lib/ReactViews/DataCatalog/DataCatalog.jsx

```diff
@@ -59,7 +58,11 @@ export const DataCatalog = observer(
               <DataCatalogMember
                 viewState={this.props.viewState}
                 member={item}
-                manageIsOpenLocally={isSearching}
+                // manage group `isOpen` flag locally if searching through models dynamically (i.e. not using catalog index)
+                // This must be false if usingCatalogIndex - so group references open correctly in the search
+                manageIsOpenLocally={
+                  isSearching && !catalogSearchProvider.usingCatalogIndex
+                }
```

Comment:
If `catalogSearchProvider` is a generic `SearchProvider` like in the previous suggestion then this flag name has to be more generic I guess, something like `shouldExpandResult`?

Reply:
Done, I added a property to `SearchProvider` called `resultsAreReferences`